lib/vector-core/src/event/vrl_target.rs

```diff
@@ -321,23 +331,22 @@ impl From<Event> for VrlTarget {
 //   * If an element is an object, create an event using that as fields.
 //   * If an element is anything else, assign to the `message` key.
 // * If `.` is anything else, assign to the `message` key.
-fn value_into_log_events(value: Value, metadata: EventMetadata) -> impl Iterator<Item = Event> {
+fn value_into_logevent(value: Value, metadata: EventMetadata) -> impl Iterator<Item = LogEvent> {
```

Comment:
Was this renaming intentional? It implies singular, but returns an iterator of `LogEvent`.

Reply:
It was indeed intentional as an attempt to make the new return type, `LogEvent` instead of `Event` explicit, though I should have kept the plural.